devtools/conda-recipe/meta.yaml

```diff
@@ -28,8 +28,8 @@ requirements:
     - matplotlib
     - seaborn
     - mdtraj
-    - parmed # for testing/debugging
-    #- pdbfixer
+    - parmed
```

Comment:
Do you need the latest parmed, or could the older parmed that comes with `ambertools` be OK?

Reply:
Also, you have `parmed` on line 23 as well.

Reply:
You can also remove `cuda92` from line 21

Reply:
I need the latest parmed. How should I specify it?

Reply:
For now, I'd keep `parmed` but list it last. Not sure if order helps, but it'll remind me.

Reply:
Ok, done.